lib/gooddata/lcm/actions/collect_segments.rb

```diff
@@ -20,6 +20,12 @@ class CollectSegments < BaseAction
 
         description 'Segments to provision'
         param :segments_filter, array_of(instance_of(Type::StringType)), required: false
+
+        description 'DataProduct'
+        param :data_product, instance_of(Type::GdProductType), required: false
+
+        description 'DataLogger'
+        param :gdc_logger, instance_of(Type::GdLogger), required: true
```

Comment:
I'd hate to be overly semantical, but this is probably just a "logger", it does not log data